### ISSUE TITLE
CLI fails with IsADirectoryError on "rio_tiler/cmap_data"

### ISSUE THREAD

**Issue by erhhung** [developmentseed/cogeo-mosaic#170]
Just installed the CLI according to directions, but it fails to run for me:
```
$ pip install cogeo-mosaic --pre
...
$ cogeo-mosaic

Traceback (most recent call last):
  File "/usr/local/bin/cogeo-mosaic", line 5, in <module>
    from cogeo_mosaic.scripts.cli import cogeo_cli
  File "/usr/local/lib/python3.9/site-packages/cogeo_mosaic/scripts/cli.py", line 14, in <module>
    from cogeo_mosaic.backends import MosaicBackend
  File "/usr/local/lib/python3.9/site-packages/cogeo_mosaic/backends/__init__.py", line 6, in <module>
    from cogeo_mosaic.backends.base import BaseBackend
  File "/usr/local/lib/python3.9/site-packages/cogeo_mosaic/backends/base.py", line 12, in <module>
    from rio_tiler.constants import WEB_MERCATOR_TMS
  File "/usr/local/lib/python3.9/site-packages/rio_tiler/__init__.py", line 3, in <module>
    from . import (  # noqa
  File "/usr/local/lib/python3.9/site-packages/rio_tiler/colormap.py", line 27, in <module>
    with pkg_resources.path(__package__, "cmap_data") as p:
  File "/usr/local/Cellar/python@3.9/3.9.2_4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/contextlib.py", line 117, in __enter__
    return next(self.gen)
  File "/usr/local/Cellar/python@3.9/3.9.2_4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/resources.py", line 175, in _path_from_reader
    opener_reader = reader.open_resource(norm_resource)
  File "<frozen importlib._bootstrap_external>", line 995, in open_resource
IsADirectoryError: [Errno 21] Is a directory: '/usr/local/lib/python3.9/site-packages/rio_tiler/cmap_data'
```
Is it not compatible with the `rio_tiler` I have installed (v2.0.6)?

**Comment by vincentsarago** [comment]
@erhhung thanks for reporting this. This is a rio-tiler issue... but I can't reproduce it locally.

We are not yet testing rio-tiler for python 3.9 yet, but I'll work on this today. to solve your issue for now you'll have to pin rio-tiler==2.0.5 for now

**Comment by vincentsarago** [comment]
@erhhung this is fixed now in rio-tiler 2.0.7 https://github.com/cogeotiff/rio-tiler/blob/master/CHANGES.md#207-2021-04-01
